fw/http_limits.c

```diff
@@ -655,9 +655,11 @@ frang_http_domain_fronting_check(const TfwHttpReq *req, FrangAcc *ra)
 		frang_msg("vhost by SNI doesn't match vhost by authority",
 			  &FRANG_ACC2CLI(ra)->addr, " ('%.*s' vs '%.*s')\n",
 			  PR_TFW_STR(&tls_name), PR_TFW_STR(&req_name));
+		tfw_vhost_put(tls_vhost);
 		return TFW_BLOCK;
 	}
 
+	tfw_vhost_put(tls_vhost);
```

Comment:
Oh, I did miss this during review. Could you please add comments to `__lookup_vhost_by_authority()` and/or other vhost lookup functions that it's a caller responsibility to put vhost?

Reply:
Yes fixed